#! /bin/sh /usr/share/dpatch/dpatch-run
## 03_electricsheep.c_DATADIR.dpatch by Alexis Sukrieh <sukria@sukria.net>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Change the splash_prefix to /usr/share/electricsheep

@DPATCH@
diff -urNad electricsheep-2.6.8~/electricsheep.c electricsheep-2.6.8/electricsheep.c
--- electricsheep-2.6.8~/electricsheep.c	2006-10-11 07:56:04.000000000 +0200
+++ electricsheep-2.6.8/electricsheep.c	2006-10-11 07:56:15.000000000 +0200
@@ -56,7 +56,7 @@
 #define STATFS statfs
 #endif
 
-char *splash_prefix = PACKAGE_DATA_DIR "/electricsheep";
+char *splash_prefix = "/usr/share/electricsheep/electricsheep";
 
 #define uniqueid_len 16
 char uniqueid[uniqueid_len+1];
